reference/entitlement.v1.yaml

```diff
@@ -0,0 +1,245 @@
+openapi: 3.0.0
+info:
+  title: entitlement.v1
+  version: '1.0'
+  description: |-
+    This is an API provided by the Marketplace in the SEM ecosystem, and is designed to enable a marketplace to make information about products that have been selected or purchased by a school or individual to be then added automatically into the LMS of the school, as well as be used by the specific Learning Application that has been purchased to ensure that all setup and provisioning has been completed in advance of access.
+
+    Events generated by the Entitlement API are connected to state changes.
+
+     - `mp.entitlement.active` - First created and active
+     - `mp.entitlement.refunded` - Refunded
+     - `mp.entitlement.updated` - Modified in any other way
+servers:
+  - url: 'https://example.stichtingsem.org/api'
+    description: entitlement
+paths:
+  '/entitlements/{id}':
+    parameters:
+      - schema:
+          type: string
+        name: id
+        in: path
+        required: true
+        description: Specific identifier of an entitlement.
+    get:
+      summary: Get a specific entitlement
+      tags: []
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Entitlement'
+      operationId: get-entitlements-by-id
+      description: 'Retrieve the full specification for an entitlement from a marketplace, needs to confirm that the caller is able to access either based on their vendor information (they can only see their own products) in the case of the vendor scope.'
+      security:
+        - API Key - 1:
+            - mp.entitlements.school
+            - mp.entitlements.vendor
+  /entitlements/vendor:
+    parameters: []
+    get:
+      summary: Get all entitlements for a specific vendor (Learning Application Provider)
+      tags: []
+      responses:
+        '200':
+          description: OK
+      operationId: get-entitlements-vendor-id
+      security:
+        - API Key - 1:
+            - mp.entitlements.vendor
+      description: 'Get all entitlements related to products from a specific vendor.  It is assumed that at the point of setup and provision of an API key, that the Vendor name is also stored against the API key by the marketplace, so that all calls with this key will be filtered to that specific vendor in the response.'
+  '/entitlements/school/{id}':
```

Comment:
would be nice if you can filter on active ones. 
'/entitlements/school/{id}/state/{status}'

Reply:
This has now been added, along with pagination params.